### ISSUE TITLE
Outbox shown too prominently

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/mail#7694]
### Is your feature request related to a problem? Please describe.

The "Outbox" shown permanently is quite confusing and technical, especially also because it is not common from other mail apps.
I catch myself that when I send an email I always wait for the (1) counter on the Outbox to go away until I do anything else (like deleting the email I responded to). And that kind of defeats the purpose of the inline sending/undo.

### Describe the solution you'd like

What was suggested by @small1 is to only use the Drafts and Sent folder for the flow and not have Outbox – and I agree with that.

It simplifies the mental model a lot as then a mail you are writing/sending can only be in 2 places (Drafts or Sent) instead of 3 (Outbox additionally).

If there’s a mail in scheduled to be sent or an issue with a mail supposed to be sent, the Drafts folder could have the counter.

@GretaD as discussed.

### Additional context

Here is how Outlook and Gmail do it.
Gmail has a "Scheduled" entry but most of the time it’s empty anyway.

Outlook | Gmail
-|-
![20221128_143257](https://user-images.githubusercontent.com/925062/204503498-baf122c8-c854-410f-8952-7eee03e40ade.png)|![image](https://user-images.githubusercontent.com/925062/204503510-acc2c39d-7057-41da-9ac6-cac5a728302a.png)

**Comment by ChristophWurst** [comment]
As a first step we will show/hide the outbox only when it has contents.

**Comment by jancborchardt** [comment]
@GretaD so @ChristophWurst and I talked about this a bit (sorry, you were fast on the pull request :D):
- It’s not such a good idea to hide and show the Outbox depending on contents, since then the folder list on the left will move around often (always when sending mail, showing for 7 seconds, then going away.
- We discussed an alternative of showing/hiding and leaving the vertical space empty when it’s hidden, but that just looks off. Even with space reduced it looks weird, and even worse when the Outbox actually shows

A possible approach would be to show the Outbox (while it has contents) sticky on the bottom. This has the challenge that we need to separate it a bit from the other list items if the list is long – this is what the divider line is for. The divider line only needs to show when the Outbox shows, not when only the Mail settings show.
Maybe there’s also a nicer way to do this with a fade to transparency of the scrollable content list? But not entirely sure how to do that in CSS. Maybe it’s also enough to just have the cutoff.
With divider | With fade to transparency
-|-
![image](https://user-images.githubusercontent.com/925062/204825798-9cdb56c6-16eb-4c43-960f-c16b166598fb.png)|![Mail outbox](https://user-images.githubusercontent.com/925062/204828986-2b3b0e4a-cfa1-47d9-b522-3a214c2efb91.png)
